neqo-transport/src/connection/mod.rs

```diff
@@ -2471,14 +2476,7 @@ impl Connection {
             }
             Frame::StreamsBlocked { stream_type, .. } => {
                 self.stats.borrow_mut().frame_rx.streams_blocked += 1;
-                let local_max = match stream_type {
-                    StreamType::BiDi => &mut self.indexes.local_max_stream_bidi,
-                    StreamType::UniDi => &mut self.indexes.local_max_stream_uni,
-                };
-
-                self.flow_mgr
-                    .borrow_mut()
-                    .max_streams(*local_max, stream_type)
+                self.remote_streams_fc[stream_type].send_flowc_update();
```

Comment:
I don't think that we need to include this call here.  We probably never needed the original code either.

Every time we retire a stream we already send an update.  So this isn't going to do anything.  Maybe replace this with a comment say that much instead.